recipes/Diamond/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+conda install boost
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX/bin ..
```

Comment:
Won't solve the boost problems, but the prefix should be $PREFIX, not $PREFIX/bin